.travis.yml

```diff
@@ -0,0 +1,12 @@
+branches:
+  except:
+    - master
+language: ruby
+cache: bundler
+sudo: false
+rvm:
+- 2.1.6
```

Comment:
So you have 2.1.6 here... current version of chefdk ships 2.1.8... but your Gemfile doesn't install on either of these: 

    ...
    Using ruby_dep 1.3.1
    Installing listen 3.1.5

    Gem::InstallError: listen requires Ruby version >= 2.2.3, ~> 2.2.
    An error occurred while installing listen (3.1.5), and Bundler cannot continue.
    Make sure that `gem install listen -v '3.1.5'` succeeds before bundling.